database/migrations/2022_10_20_182207_create_times_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('times', function (Blueprint $table) {
+            $table->id();
+            $table->string('name', 50);
+            $table->string('date', 50);
+            $table->string('time', 50);
+            // $table->timestamps();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('times');
+    }
+};
```

Comment:
You need to add data and time columns in `tasks` table no need to create new table